src/main/java/com/google/sps/servlets/ScheduleServlet.java

```diff
@@ -36,15 +37,29 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     CalendarInterface calendarInterface = new CalendarInterface();
     List<Event> calendarEvents = calendarInterface.loadPrimaryCalendarEventsOfTomorrow();
```

Comment:
You should load the events of the day in which you want to schedule.

Reply:
Ohh uhh

Reply:
oh good catch!